src/hotel.js

```diff
@@ -1,14 +1,17 @@
 import $ from 'jquery';
+import Customer from './customer.js'
 
 class Hotel {
-  constructor(users, bookings, rooms) {
-    this.users = users;
-    this.bookings = bookings;
+  constructor(users, rooms, bookings, date) {
+    this.allUsers = users;
     this.rooms = rooms;
-    // this.today = this.getTodaysDate();
-    this.currentUser;
-    this.date;
+    this.bookings = bookings;
```

Comment:
Good job building out a class to hold all of the data you are fetching.

Reply:
Thank! After giving it some thought I decided all rooms, users, and bookings should live on the hotel class. I was also debating if the customer class should hold onto the current date, or should it be kept in the hotel class. @pJanks What do you think?